### PR TITLE
Add QM position

### PR DESCRIPTION
**Pull request by davidlmobley**
@mrshirts can you please review this at the same time? QM job ad has been edited by various people so it's just "check that the change to the website looks OK".

### CODE REVIEW

File: content/about/careers/openff/_index.md [hunk 1]
```diff
@@ -5,6 +5,52 @@ weight: 1
...
+
+**About us**: We are a distributed team of PhD students, postdocs and software scientists led by [5 academic PIs -- Mike Gilson, David Mobley, Lee-Ping Wang, Michael Shirts, and John Chodera](https://openforcefield.org/about/team/), with more direct leadership from technical lead Jeff Wagner (OMSF) and science lead Lily Wang (OMSF). The effort has been primarily funded by the Open Force Field Consortium and an NIH grant focused on development of accurate biomolecular force fields. The [Open Force Field Consortium](https://openforcefield.org/about/organization/) is an [alliance of pharmaceutical and biotech companies](https://openforcefield.org/about/organization/#open-force-field-consortium) aiming to advance the state of force fields for biomolecular simulation and design through an open collaborative effort. In our organization, the postdocs and PhD students do research and exploratory work using our tools/data, while a team of software and staff scientists (holding PhDs in biomolecular research) work on taking new innovations to production level via high quality professional software and force fields.
```
Comment (mrshirts) [inline content/about/careers/openff/_index.md:22]: Lee-Ping Wang is listed here as a site, do we want that?

File: content/about/careers/openff/_index.md [hunk 2]
```diff
@@ -5,6 +5,52 @@ weight: 1
...
+
+**Location and employer**
+The candidate will be employed by the Open Molecular Software Foundation ([OMSF](https://omsf.io), see https://omsf.io), a 501c3 nonprofit organization which hosts OpenFF and several other open source projects in the molecular modeling space, including Open Free Energy, OpenFold, and WESTPA, among others. OMSF is a fully remote organization, so employees can be located essentially anywhere in the world. However, if you prefer working in a physical office, that may also be possible, as we have relationships with a number of potential host institutions where a Visiting Scientist appointment (or similar) may be possible.
```
Comment (mrshirts) [inline content/about/careers/openff/_index.md]: Should it be 501(c)(3)?

**Review by mrshirts — COMMENTED** [review]
See comments in files.

File: content/about/careers/openff/_index.md (hunk 1 above)
Comment (davidlmobley) [reply content/about/careers/openff/_index.md:22]: At this point yes, will remove if he doesn't want to be listed (we are discussing).

File: content/about/careers/openff/_index.md (hunk 2 above)
Comment (davidlmobley) [reply content/about/careers/openff/_index.md]: sure

**Pull request merged**